roles/aci/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# This role contains tasks specific to aci
+
+  - name: Install docker-py as a workaround for Ansible issue
```

Comment:
The issue is that docker-py is needed for docker. Without it, docker module returns random errors. Found this from the github discussion on the topic.